content-categories.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+$categories = get_categories(); ?>
+
+<ul class="nav nav-pills" role="tablist">
+	<li class="active">
+		<a  href="#records-research" role="tab" data-toggle="pill">Records and research</a>
+	</li>
+	<li>
+		<a href="#behind-the-scenes" role="tab" data-toggle="pill">Behind the scenes</a>
+	</li>
+	<li>
+		<a href="#technology-innovation" role="tab" data-toggle="pill">Technology and innovation</a>
+	</li>
+	<li>
+		<a href="#managing-information" role="tab" data-toggle="pill">Managing information</a>
+	</li>
+	<li>
+		<a href="#archives-and-archivists" role="tab" data-toggle="pill">Archives and archivists</a>
+	</li>
+</ul>
+<div class="tab-content clearfix">
+<?php foreach ( $categories as $cat ) {
```

Comment:
Possible the whole thing into a function? To stay away from inline PHP inside files wherever's possible?

Reply:
Yes, we can at a later stage